### ISSUE TITLE
Add unit test for NNs we'll need in ATLAS code

### ISSUE THREAD

**Issue by dguest** [lwtnn/lwtnn#31]
We'll be adding several taggers to an Athena release soon, so it would be useful to write a unit test around a representative set of them. We currently already have something for `ipmp` which covers a good fraction of the code (for GRUs, highway layers, and maxout), but it would be nice to add something _close to_ the DL1 implementation we end up using.

**Comment by jwsmithers** [comment]
Before I submit a pull request, let me know if my logic is flawed somewhere here:

1) Using arch, variable and weight file from the [jira ticket](https://its.cern.ch/jira/browse/AFT-218), I cloned your [test repository](https://github.com/dguest/lwtnn-test-data) and put DL1.tgz there (submitted merge request for that, [here](https://github.com/dguest/lwtnn-test-data/pull/1)).

2) I then copied the ipmp test and turned it into a DL1 test

3) Then, locally on my laptop I used the `lwtnn-test-keras-arbitrary-net.py` script, along with the inputs from DL1.tgz to get outputs for u,b and c jets

4) Then I put those into the `/tests/data` and ran the test script

And there is agreement. Does that all sound right?
